meetings/2021/20210913-website.md

```diff
@@ -0,0 +1,134 @@
+
+#### Job Board
+
+- A job board might be interesting to build to help the community find jobs
+- A job board could also be monetized to help fund future projects
+
+#### Learning Platform
+
+- A learning platform was discussed as a possible offering to give back to the community
+
+#### Gitcoin
+
+- Moving things to Gitcoin was discussed to make projects more decentralized
+
+### Social Media
+
+- Currently the DAO only uses Twitter and Discord for communication and exploration of other avenues / mediums should be looked at
+
+#### Twitter
+
+- Unconfirmed: `@nader#1506` is the sole person who controls [@developer_dao](https://twitter.com/developer_dao], and we would like to discuss how that account can be utilized by the community to give updates
+
+#### Blogging
+
+- [Medium.com](https://medium.com) was presented as an option to blog on
+- [Dev.to](https://dev.to) a more dev focused blogging platform
+- [Mirrow.xyz](https://mirrow.xyz) a more web3 focus blogging platform
+- [Hashnode](https://hashnode.com) another blog platform
+- [Ghost](https://ghost.org) a possible solution for self hosting
+- Undecided was how the submissions process would work, if any, and if there would be a moderation process put in place as well
+- `@codingwithmanny#0205` mentioned that we should leverage existing blog platforms because that is naturally where readers go for content, which would increase exposure
+- `@keretos#6430` mentioned that we should start thinking about blogging when we have a reason to blog and may consider shelving this for now
+
+### Misc
+
+- Additional topics that may not be directly related but have some relevance
+
+#### MEE6
+
+- `@kempsterrrr#8590` discussed the idea of having an onboarding channel that would be slightly automated with MEE6, where we could gather information about skills, and incentivize new users to follow social media accounts with a reward as a quick win
+- There might be some integration with the website on how to be onboarded as a section
+
+#### Becoming More Decentralized
+
+- `@Raz#1337` mentioned [Gitcoin](https://gitcoin.co/how/funder) for storing our code, [Poap](https://poap.xyz/faqs) for tracking participation, [Element.io](https://element.io) as an alternative to Discord
+- Concern was risen that these additional apps could increase the barrier of entry
+
+### Closing
+
+- Great meeting
+- There will be a separate conversation about governance and we'll rely on `@willblackburn#5859`'s experience
+- Current process for features should be in Github issues for now
+
+# 📝 Notes
+
+### 📋 Attendees
+
+Too many to name at the moment
+
+### ✅ Action items
+
+1. Need someone to take hold of the Gitcoin POC
+2. Decision on if we should create a blog or post on an existing platform
+3. Create a repository for meeting minutes
+
+### 📹 Video recording
+
+- No video recording was created at this time (possible future idea)
```

Comment:
Love this idea. Would make it so those that can't attend can get the full context of the meeting if they like

Reply:
Yeah I was thinking perhaps if we have a YouTube channel like the NodeJS foundation and their meetings.
https://www.youtube.com/c/nodejs-foundation/videos